RetailCoder.VBE/Refactorings/Rename/RenameModel.cs

```diff
@@ -76,5 +108,41 @@ public void PromptIfTargetImplementsInterface(ref Declaration target)
 
             target = interfaceMember;
         }
+
+        internal class Rewriters
```

Comment:
ref. http://chat.stackexchange.com/transcript/message/36380190#36380190 - that internal class should be going away.